pages/api/login.js

```diff
@@ -24,6 +24,7 @@ async function handler(request, response) {
       first_name: user.first_name,
       last_name: user.last_name,
       registered_at: user.registered_at,
+      telephone: user.telephone,
```

Comment:
```suggestion
      phone: user.telephone,
```

Our default property is called `phone` which is automatically synced to destinations ^^